action.yml

```diff
@@ -7,7 +7,16 @@ branding:
 inputs:
   project:
     required: true
+    description: The number of the project which the issues will be synced to
     type: number
+  project_field:
```

Comment:
These are named in 2 different ways - `project fields`, and `custom fields`.
Consider renaming something in order to stick to 1 name.